### PR TITLE
add update-images.sh for docker-host

### PR DESCRIPTION
**Pull request by xantrix**
Linked PR in Lagoon
https://github.com/amazeeio/lagoon/pull/2368

Closes: #151

### CODE REVIEW

File: charts/lagoon-remote/values.yaml [hunk 1]
```diff
@@ -29,7 +29,7 @@ dockerHost:
 
   registry: registry.lagoon.svc:5000
 
-  repositoryToUpdate: lagoon
```
Comment (smlx) [inline charts/lagoon-remote/values.yaml:32]: I think this change might not work.

Looking at the script in https://github.com/amazeeio/lagoon/pull/2368 it greps for `${REPOSITORY_TO_UPDATE}/`. Our base image repositories are [`testlagoon`](https://hub.docker.com/u/testlagoon) and [`uselagoon`](https://hub.docker.com/u/uselagoon), so having `amazeeio` here will mean that the base images don't get updated. Could you confirm that understanding of where the base images are pushed @tobybellwood?

Comment (tobybellwood) [reply charts/lagoon-remote/values.yaml:32]: It's an -E grep, so it probably should cover all our repos? 
`amazeeio|uselagoon` for production clusters, and add `testlagoon` if there's going to be test workloads on it?

`amazeeio|lagoon` would also cover it

Comment (xantrix) [reply charts/lagoon-remote/values.yaml:32]: thanks for clarifying @tobybellwood 
in our production cluster, I saw `amazeeio` as base image repo.
If we agree I'd proceed using `amazeeio|lagoon`

Comment (smlx) [reply charts/lagoon-remote/values.yaml:32]: sounds good :+1:

**Comment by smlx** [comment]
hi @xantrix sorry this has languished a bit.. could you rebase onto main and we'll get this in ASAP.

**Comment by xantrix** [comment]
no worries @smlx :) rebased

**Review by smlx — APPROVED** [review]
LGTM

**Pull request merged**